lib/id3c/cli/command/etl/fhir.py

```diff
@@ -223,6 +227,11 @@ def process_diagnostic_report_bundle_entry(db: DatabaseSession, bundle: Bundle,
         if not is_collection_identifier and not sample:
             raise SampleNotFoundError("No sample with identifier «{specimen_identifier.uuid}» found.")
 
+        if specimen:
+            # If there is a Specimen Resource, check for potential updates
+            # that need to be handled by FHIR ETL customizations.
+            sample = INTERNAL_CUSTOMIZATIONS.check_specimen_for_update(db, sample, specimen)
```

Comment:
Now that I'm looking at this again, it feels weird that I'm calling a custom function within core. Is there another way of doing this?